src/main/java/com/conveyal/datatools/manager/controllers/api/UserController.java

```diff
@@ -42,10 +42,7 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.Date;
-import java.util.Iterator;
 import java.util.List;
-import java.util.Map;
-
```

Comment:
New line between native package imports and static imports.

Reply:
Updated.